src/main/scala/org/zalando/spearheads/innkeeper/dao/PathsRepo.scala

```diff
@@ -11,4 +15,5 @@ trait PathsRepo {
   def selectAll: DatabasePublisher[PathRow]
   def selectByOwnerTeamAndUri(ownedByTeam: Option[String], uri: Option[String]): DatabasePublisher[PathRow]
   def pathWithUriExists(uri: String): Future[Boolean]
+  def patch(id: Long, pathPatch: PathPatch, updatedAt: LocalDateTime): Future[Option[PathRow]]
```

Comment:
The names in the Repos should be in the database domain and not http domain. I would rename this to update.